### PR TITLE
Implemented resume function

### PR DESCRIPTION
**Pull request by kwillno**
I implemented a resume function for the CLI.  
The function is based largely on the `start` function, but uses the description from the previously logged record. From my testing it behaves the same as `start`. If no records are found earlier in the day, an error message is given and functions returned.

### CODE REVIEW

File: timetagger_cli/core.py [hunk 1]
```diff
@@ -154,6 +154,70 @@ def stop():
...
+        print("No records earlier today.")
+        return 
+
+    # Remove HIDDEN records
+    filtered_records = []
+    for r in records:
+        if "HIDDEN" in r["ds"]:
+            continue
+
+        filtered_records.append(r)
```
Comment (almarklein) [inline timetagger_cli/core.py:182]: Great place for a list comprehension:
```suggestion
    filtered_records = [r for r in records if "HIDDEN" not in r["ds"]]
```

File: timetagger_cli/core.py [hunk 2]
```diff
@@ -154,6 +154,70 @@ def stop():
         print_records(running_records)
 
 
+def resume():
+    """Start a timer with the same settings as previous"""
```
Comment (almarklein) [inline timetagger_cli/core.py:158]: A bit more precise:
```suggestion
    """Start a timer with the same description as previous."""
```

**Review by almarklein — COMMENTED** [review]
Thanks! This looks great. I only have two minor suggestions.

**Comment by almarklein** [comment]
Oh, it also looks like `black` has a new version with some new rules. You can fix them by running `black .`  with a recent version of `black`.  I can also do it in a new pr if you prefer.

**Comment by kwillno** [comment]
I can try to fix the `black` thing.  
Added your suggestions! Thanks for the time :D

**Comment by almarklein** [comment]
Made a new release too.

**Comment by martin-braun** [comment]
Question: Why is there an error message if there is no task on the same day? I would prefer it would repeat the last task no matter how far away. Sometimes I work on huge tasks that spread through multiple days.

I would assume resuming a task behaves the same way like in the web version at least, meaning it would repeat the last task, no matter how old it is. Thoughts?

**Comment by almarklein** [comment]
Good point. I can indeed imagine starting in the morning and wanting to resume what I was doing yesterday afternoon.

Even nicer would be to show multiple options (defaulting to the last).

**Comment by kwillno** [comment]
Ahh, sorry @martin-braun  
I did not realise that is the way it functions in web version. I still think that the best solution is the one proposed earlier though, where we can choose between a few options.

**Pull request merged**